### PR TITLE
CS fix

### PR DESCRIPTION
**Pull request by OskarStark**
#104 is failing due to the missing CS fixes

I applied them to `1.x` instead, shall I create a separate PR against `master` or do we fix that by merging `1.x` -> `master`?

### CODE REVIEW

**Comment by greg0ire** [comment]
I applied them directly on the dev-kit branch, but your method should work too. The thing is, we also need to remove the StyleCI bridge so we need to push to the dev-kit branch anyway

**Comment by OskarStark** [comment]
So shall we close this PR? Or shall I remove the bridge in my PR?

File: Tests/Filter/DateFilterTest.php [hunk 1]
```diff
@@ -20,7 +22,7 @@
  */
 class DateFilterTest extends TestCase
 {
-    public function testGetFilterTypeClass()
+    public function testGetFilterTypeClass(): void
```
Comment (greg0ire) [inline Tests/Filter/DateFilterTest.php:25]: You need to avoid this by removing all the php IFs but the 5.6 one in `.php_cs.cache`

**Review by greg0ire — CHANGES_REQUESTED** [review]

File: Tests/Filter/DateFilterTest.php (hunk 1 above)
Comment (OskarStark) [reply Tests/Filter/DateFilterTest.php:25]: I don't really get what you mean 😄

Comment (greg0ire) [reply Tests/Filter/DateFilterTest.php:25]: Sorry wrong file, I meant `.php_cs.dist`. This one possibly only appears on the dev-kit branch

Comment (greg0ire) [reply Tests/Filter/DateFilterTest.php:25]: cc @Soullivaneuh

**Comment by jordisala1991** [comment]
This PR needs to be applied on master only

**Comment by jordisala1991** [comment]
IMO the best way is to do it on the master-dev-kit PR

**Comment by OskarStark** [comment]
ok, lets close it

**Pull request closed without merging**